pkg/plugin/plugin.go

```
@@ -151,6 +151,22 @@ func splitIDs(s string) []string {
 	return ids
 }
 
+// resolveTimeout returns the query timeout to apply: the one configured on the
+// datasource when it is set and parses, otherwise the supplied default.
+func resolveTimeout(raw string, def time.Duration) time.Duration {
```

golangci-lint (2.12.2)

[error] func resolveTimeout is unused

(unused)